website/content/gateway/api_reference/resources/debits/recipient_information.md

```diff
@@ -23,6 +23,11 @@ Example: "Doe Jane A." (last name, first name, optional middle initial).
 {{% regex_optional %}}Required.{{% /regex_optional %}}
 {{% /description_details %}}
 
+{{% description_term %}}recipient[account_number_type] {{% regex %}}(other|rtn_and_ban|iban|pan|email|phone_number|ban_and_bic|wallet_id|social_network_id){{% /regex %}}{{% /description_term %}}
```

Comment:
Could it make sense to have `recipient[account_number_type]` before `recipient[account_number]`?

Reply:
Nah, let's just go with the current order.